amperity_reference/source/workflows.rst

```diff
@@ -702,13 +702,45 @@ Databases
 
 Some common workflow actions that your brand may see when generating databases include:
 
+* :ref:`Cannot bind inputs to resolved state <workflows-databases-inputs-unbound>`
 * Errors resolving column names in tables
 * Missing tables
 * SQL query alerts (including warning *and* failure thresholds)
 
 .. workflows-context-databases-end
 
 
+.. _workflows-databases-inputs-unbound:
+
+Cannot bind inputs to resolved state
+++++++++++++++++++++++++++++++++++++++++++++++++++
+
+.. workflows-databases-inputs-unbound-start
+
+Inputs to the database build process, such as source and custom domain tables, stitched tables, and core tables, must have a schema that matches the one that is expected by the database.
```

Comment:
I think of "stitched" as being the _inputs_ to stitch, which by definition are some combination of source and/or custom domain tables. I usually refer to the _outputs_ of stitch as "stitch tables" or "stitch output tables", but I'm not sure what we typically say in our public-facing docs.

```suggestion
Inputs to the database build process, such as source and custom domain tables, stitch output tables, and core tables, must have a schema that matches the one that is expected by the database.
```